GhidraDocs/InstallationGuide.html

```diff
@@ -1,581 +1,672 @@
-    On the up side, administrative privilege is not required to install Ghidra for personal use.
-    Also, because installing Ghidra does not update any OS configurations such as the registry on
-    Windows, removing Ghidra is as simple as deleting the Ghidra installation directory.  On the
-    down side, Ghidra will not automatically create a shortcut on the desktop or appear in
-    application start menus.
-  </li>
-  <li>
-    Administrative privilege may be required to extract Ghidra to certain filesystem destinations
-    (such as C:\), as well as install the Ghidra Server as a service.
-  </li>
-  <li>
-    Ghidra relies on using directories outside of its installation directory to manage both
-    temporary and longer-living cache files.  Ghidra attempts to use standard OS directories that
-    are designed for these purposes in order to avoid several issues, such as storing large amounts
-    of data to a roaming profile.  If it is suspected that the default location of these directories
-    is causing a problem, they can be changed by modifying the relevant properties in the
-    <i>support/launch.properties</i> file.
-  </li>
-</ul>
-<h3><a name="JavaNotes"></a>Java Notes</h3>
-<ul>
-  <li>
-    Ghidra requires a <a href="#Requirements">supported</a> version of a Java Runtime and 
-    Development Kit on the PATH to run.  However, if there is a version of Java on the PATH that 
-    Ghidra does not support, it will use that version of Java (if 1.7 or later) to assist in 
-    locating a supported version on your system.  If one cannot be automatically located, the user 
-    will be prompted to enter a path to the Java home directory to use (the Java home directory is 
-    the parent directory of Java's <i>bin</i> directory).  This minimizes the impact Ghidra has on 
-    pre-existing configurations of Java that other software may rely on.
-  </li>  
-  <li>
-    If Ghidra failed to run because <i>no versions</i> of Java were on the PATH, a 
-    <a href="#Requirements">supported</a> JDK should be manually installed and added to the 
-    PATH.  The following steps outline how to add a JDK distribution to the operating system's 
-    PATH.
-    <ul>
-      <li>
-        <p><b>Windows: </b>Extract the JDK distribution (.zip file) to your desired location 
-        and add the JDK's bin directory to your PATH:</p>
-        <ol>
-          <li>
-            <p>Extract the JDK:</p>
-            <ol>
-              <li>Right-click on the zip file and click <b>Extract All...</b></li>
-              <li>Click <b>Extract</b></li>
-            </ol>
-          </li>
-          <li>
-            <p>Open Environment Variables window:</p>
+
+    <!--main role="main"-->
+    <main role="main">
+      <!-- Main jumbotron for a primary marketing message or call to action -->
+      <div class="jumbotron" style="text-align:center">
+	    <img style="width: 50%" src="images/GHIDRA_DOCS.png" alt="Ghidra" />
```

Comment:
Unfortunately like @astrelsky said we cannot accept the binary image.  So this line can come out too.

Reply:
This image is one which aleady exists in the repo. [GhidraDocs/images/GHIDRA_1.png](https://github.com/NationalSecurityAgency/ghidra/blob/master/GhidraDocs/images/GHIDRA_1.png)